tools/zetasql_helper/zetasql_helper/local_service/local_service_test.cc

```diff
@@ -89,12 +89,57 @@ TEST_F(LocalServiceTest, LocateTableRangesTest) {
 }
 
 
-TEST_F(LocalServiceTest, GetAllKeywords) {
+TEST_F(LocalServiceTest, GetAllKeywordsTest) {
```

Comment:
nit: stylewise you don't need the GetAllKeywords"Test"
just GetAllKeywords

Reply:
Have changed all of the tests